### ISSUE TITLE
Default configuration for subset of commands

### ISSUE THREAD

**Issue by sideninja** [onflow/flow-cli#135]
### Issue To Be Solved
Currently, all commands in CLI require configuration. All commands that currently require configuration just to get the network access point address should work without the configuration and get that address from the variable in the codebase.

### Suggest A Solution
We should define all the networks within the CLI codebase (this also relates to issue #125). If no configuration is present we use these values as a last resort solution to get the network address and thus not require configuration to be created with flow init. 

This way UX will be improved because a simple use case of using CLI tool is to just maybe get an account or query a block and with this solution implemented it won't be necessary to init a configuration first, which is also not a default thought process of a user, no one thinks "ohh I need to first run this command in order to execute what I want".

Priority of network value resolution will be:
- network flag within the configuration
- host flag
- flow configuration default
- codebase value default

The default value for the network should probably be emulator network, but then again that might not be the best decision as it again preconditions the emulator is running which is anyhow dependent on the configuration being initialized. It might make more sense to default it to testnet (as that is dev home).

**Comment by psiemens** [comment]
> We should define all the networks within the CLI codebase (this also relates to issue #125)

I agree with this as long as it's easy to override the values for Testnet and Mainnet. Currently it makes sense to define default public APIs for both networks, but in the future as more public access nodes come online, the user should be free to (or maybe even required to) specify the node they want to connect to.
